ibis/expr/datatypes.py

```diff
@@ -1805,7 +1805,7 @@ def can_cast_string_to_temporal(
     source: String,
     target: Union[Date, Time, Timestamp],
     value: Optional[str] = None,
-    **kwargs
+    **kwargs,
```

Comment:
not objecting, but why are these changing?

Reply:
Black changed that automatically, probably because I removed `py35` from the target-version property (pyproject.toml)